cicd-utils/info.json

```diff
@@ -29,8 +30,8 @@
           "visible": true,
           "editable": true,
           "placeholder": "Eg. 5af00c658289bb5618193f42",
```

Comment:
e.g. 5af00c658289bb5618193f42.zip

Reply:
in filepath .zip extension will not present so no need to add .zip extension